source/standards/continuous-delivery.html.md.erb

```diff
@@ -0,0 +1,70 @@
+---
+title: How to use continuous delivery (CD)
+expires: 2018-10-08
+last_reviewed_on: 2018-04-08
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+When your team uses [Continuous Delivery (CD)][] you can implement changes like new features or configuration updates quickly and safely, by adopting CD you:
+
+Enable frequent iterations - it’s cheaper and easier to deliver small changes to production. Releasing to production multiple times a day means you quickly get feedback from users.
```

Comment:
It'd be good to add a link for the 'Enable frequent iterations' and also 'Have maintainable code' to match 'Deploy low risk releases' and 'Build quality software'